ktor-http/ktor-http-cio/common/src/io/ktor/http/cio/internals/Chars.kt

```diff
@@ -66,7 +66,7 @@ internal fun CharSequence.parseHexLong(): Long {
     return result
 }
 
-internal fun CharSequence.parseDecLong(): Long {
+public fun CharSequence.parseDecLong(): Long {
```

Comment:
KDoc should be added for the new public API. @andreyaksenov could you check the KDocs?